lib/example_lib/Kconfig

```
@@ -0,0 +1,4 @@
+config EXAMPLE_LIB
+	bool "example_lib Support"
+	help
+	  This option enables the example_lib library
```

```suggestion
	  This option enables the example_lib library. This lib is used to demonstrate unit tests in ../tests/sample_test
```

Or something like that